docker-compose.yml

```diff
@@ -36,7 +34,8 @@ services:
   elasticsearch:
     build: elasticsearch
     environment:
-      - ES_JAVA_OPTS=-Xms500m -Xmx500m
+      - ES_JAVA_OPTS=-Xms640m -Xmx640m
```

Comment:
640M seemed to give a good balance of GC overhead and swapping (still swaps a lot when reindexing though)